bleach/callbacks.py

```diff
@@ -4,7 +4,11 @@
 
 def nofollow(attrs, new=False):
     href_key = (None, u'href')
-    if href_key not in attrs or attrs[href_key].startswith(u'mailto:'):
+
+    if href_key not in attrs:
+        return attrs
+
+    if attrs[href_key].startswith(u'mailto:'):
```

Comment:
Why'd you make this change? It doesn't seem like it changes the behavior at all.